docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -30,88 +30,3 @@ This limit will be applied to every container of the given component.
 +
 For `plugin` components, RAM limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-
-. Specify container memory request for components
-+
-To specify a container(s) memory request for `plugin` use the `memoryRequest` parameter:
-+
-.Specify container memory request for components
-====
-[source,yaml]
-----
-  components:
-   - name: exec-plugin
-     plugin:
-      id: eclipse/machine-exec-plugin/0.0.1
-      memoryLimit: 1Gi
-      memoryRequest: 512M
-   - name: maven
-     container:
-      image: eclipe/maven-jdk8:latest
-      memoryLimit: 512M
-      memoryRequest: 256M
-----
-====
-+
-This limit will be applied to every container of the given component.
-+
-For `plugin` components, RAM requests can be described in the plug-in descriptor file, typically named `meta.yaml`.
-+
-If they are not specified, the values are undetermined: they may or may not be inferred from the application that consumes the devfile or from Kubernetes.
-+
-
-. Specify container CPU limit for components
```

Comment:
CPU limit has been reintroduced to devfile 2.1 spec (https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html, devfile/api#131). Given that this change is on master which is currently mapping to 2.1 spec, this section should stay.

Reply:
The plugin is still there in the example, given that we plugin will be removed from the schema, we should remove all plugin definitions on this page.